code/modules/research/designs/autolathe/service_designs.dm

```diff
@@ -70,6 +70,65 @@
 	)
 	departmental_flags = DEPARTMENT_BITFLAG_SERVICE
 
+/datum/design/scissors
```

Comment:
modularize (just create and move these to `monkestation/code/modules/research/designs/autolathe/service_designs.dm`)